Symfony/CS/Tokenizer/Token.php

```diff
@@ -99,9 +100,22 @@ public function equals($other)
         $selfPrototype = $this->getPrototype();
 
         foreach ($otherPrototype as $key => $val) {
-            if ($selfPrototype[$key] !== $val) {
+            // Make sure the token has such key
```

Comment:
And we follow "imitate code", "follow convenstion" stated by symfony in our own project.
And why we follow it that way was mentioned above.